src/lib.rs

```diff
@@ -136,6 +160,7 @@ impl Kaboodle {
         });
     }
 
+    /// Disconnect this client from the mesh network.
```

Comment:
Optional: might be worth noting that the list of known peers will remain accessible after stopping—it will just become increasingly stale.